app/MMB/src/components/header/Location.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+import { Text, StyleSheet } from '@kiwicom/mobile-shared';
+
+function Location(props) {
+  return (
+    <Text style={styleSheet.city}>
+      <Translation passThrough={props.data.airport.city.name} />
```

Comment:
Shouldn't you use idx here?

Reply:
Why is Flow happy? 🤔

Reply:
flow is broken 🙈

Reply:
I saw just a few cases where Flow was actually broken in last year. The problem is usually between the keyboard and chair like in this case - our `createFragmentContainer` HOC is the reason why it doesn' work... 🤦‍♂️